File: List_Filtering.py
def filter_list(l):
    filtered_list = list([x for x in l if isinstance(x, int)])
    return(filtered_list)
print(filter_list([1, 2, 6, '456', 2, '43']))
print(filter_list([1, 2, 467, '456', 2, '0']))
print(filter_list([1, -456, 456, '456', 2, '43']))
print(filter_list(['56', 2, 456, '987897', 2, '43']))
print(filter_list(['8', 2, '456', 676, 2, '43']))

File: listFiltering.js
function filter_list(l) {
    return l.filter(j=> typeof j != 'string')
   }


console.log(filter_list(['34', 2, 1, '76', 3]))
console.log(filter_list(['3', 2, 109, '6', 67]))
console.log(filter_list(['4', -90, 1, '-6', 600876]))
console.log(filter_list(['23454', 2, 14, '76', 27]))